fixture/different.transformPath/src/a

```diff
@@ -0,0 +1 @@
+a
```

Comment:
All files should in an empty newline. Same with the other files.

<img width="160" alt="screen shot 2017-02-16 at 23 54 14" src="https://cloud.githubusercontent.com/assets/170270/23031350/4088a282-f4a3-11e6-94e1-1251340a54be.png">